### PR TITLE
lowercase AND and keep the comma

### PR DESCRIPTION
**Pull request by ghost**
typo

### CODE REVIEW

File: files/en-us/web/css/border/index.html [hunk 1]
```diff
@@ -91,7 +91,7 @@ <h3 id="Setting_a_pink_outset_border">Setting a pink outset border</h3>
 
 <h4 id="HTML">HTML</h4>
 
-<pre class="brush: html notranslate">&lt;div&gt;I have a border, an outline, AND a box shadow! Amazing, isn't it?&lt;/div&gt;
+<pre class="brush: html notranslate">&lt;div&gt;I have a border, an outline and a box shadow! Amazing, isn't it?&lt;/div&gt;
```
Comment (ericwbailey) [inline files/en-us/web/css/border/index.html:94]:
```suggestion
<pre class="brush: html notranslate">&lt;div&gt;I have a border, an outline, and a box shadow! Amazing, isn't it?&lt;/div&gt;
```

Oxford comma clarifies the `div`'s three properties.

**Review by ericwbailey — CHANGES_REQUESTED** [review]

File: files/en-us/web/css/border/index.html (hunk 1 above)
Comment (mattiapontonio) [reply files/en-us/web/css/border/index.html:94]: I could commit but really?
Can You please link something?

Comment (ericwbailey) [reply files/en-us/web/css/border/index.html:94]: https://www.nytimes.com/2018/02/09/us/oxford-comma-maine.html

**Comment by mattiapontonio** [comment]
That's true 😲

**Review by ericwbailey — APPROVED** [review]
>We should keep this in mind in some issue in translation? 🤔
>developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat/ListFormat

Definitely something to think about, but this one is good to go.

**Pull request merged**